api/src/test/java/io/strimzi/api/kafka/model/KafkaUserCrdIT.java

```diff
@@ -49,7 +49,7 @@ void testKafkaUserWithExtraProperty() {
             KubeClusterException.class,
             () -> createDeleteCustomResource("KafkaUser-with-extra-property.yaml"));
 
-        assertThat(exception.getMessage(), containsString("unknown field \"thisPropertyIsNotInTheSchema\""));
+        assertThat(exception.getMessage(), containsString("unknown field \"spec.thisPropertyIsNotInTheSchema\""));
```

Comment:
Same as above. This seems quite strange as I would not expect this to change. What version of Kubernetes did you run the tests against?

Reply:
I'm using v1.25.0 to test.